person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
```

Comment:
- You should also add the setter/getter method for the name and age as this would be used to check the person's age if they have the parent's permission and also, their name `attr_accessor :name, :age`